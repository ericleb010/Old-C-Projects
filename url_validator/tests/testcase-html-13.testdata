-d 1 http://php.net/manual/en/index.php
