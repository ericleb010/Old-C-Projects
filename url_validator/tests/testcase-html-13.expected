
Checking http://php.net/manual/en/index.php ... OK
